efficientdet/graph_editor/BUILD

```diff
@@ -0,0 +1,162 @@
+# Description:
+#   contains parts of TensorFlow that are experimental or unstable and which are not supported.
```

Comment:
Do we need Bazel to build it?

Reply:
@fsx950223 you don't need to build it for this gradient check-pointing and I never tried. Graph editor is not fully tested as a standalone tf 2.0 library but the functionality needed for gradient checkpointing works